plugins/inputs/varnish/varnish.go

```diff
@@ -139,6 +244,16 @@ func (s *Varnish) Gather(acc telegraf.Accumulator) error {
 			continue
 		}
 
+		//skip not active vcls
+		vMetric := parseMetricV2(stat)
```

Comment:
My goal was also to fix https://github.com/influxdata/EAR/issues/2289 and strip the active VCL name from field name for v1 metrics. If we want v1 to stay fully backward compatible, we should revert this and remove also unnecessary  `varnishadm vcl.list` check.

Reply:
Hi,

The issue to keep nonactive VCL's is that it can generate huge of metrics that seems useless.

It can be very good to skip all inactive vcls.

There is a similar PR available on https://github.com/influxdata/telegraf/pull/6832 (but seems abandoned) related to https://github.com/influxdata/telegraf/issues/5622

Reply:
I understand but we try to keep backwards compatibility as up until now nonactive vcl's were reported. The approach is to give users the option to opt in to new behavior rather than changing behavior on an upgrade of Telegraf.